ofrak_core/test_ofrak/unit/test_ofrak_server.py

```diff
@@ -60,40 +61,55 @@ async def test_get_index(ofrak_server, aiohttp_client):
     assert resp.headers["Content-Type"] == "text/html"
 
 
-async def test_create_root_resource(ofrak_server, aiohttp_client, hello_world_elf):
+async def test_create_root_resource(ofrak_context, ofrak_server, aiohttp_client, hello_world_elf):
     client = await aiohttp_client(ofrak_server._app)
-    resp = await client.post("/create_root_resource", params={"name": "test"}, data=hello_world_elf)
+    resp = await client.post(
+        "/create_root_resource", params={"name": "hello_world_elf"}, data=hello_world_elf
+    )
     assert resp.status == 200
     body = await resp.json()
     assert body["id"] is not None
 
+    resource = await ofrak_context.create_root_resource(hello_world_elf, hello_world_elf, (File,))
+    serialized_resource = ofrak_server._serialize_resource(resource)
+    json_result = json.loads(json.dumps(serialized_resource))
+    assert body["tags"] == json_result["tags"]
 
-async def test_get_root_resources(ofrak_server, aiohttp_client, hello_world_elf):
+
+async def test_get_root_resources(ofrak_context, ofrak_server, aiohttp_client, hello_world_elf):
     client = await aiohttp_client(ofrak_server._app)
     create_resp = await client.post(
-        "/create_root_resource", params={"name": "test"}, data=hello_world_elf
+        "/create_root_resource", params={"name": "hello_world_elf"}, data=hello_world_elf
     )
     get_resp = await client.get("/get_root_resources")
     assert get_resp.status == 200
     body = await get_resp.json()
     assert body[0]["id"] is not None
 
+    resource = await ofrak_context.create_root_resource(hello_world_elf, hello_world_elf, (File,))
```

Comment:
One thing I would recommend is to make both `client` and `resource` into fixtures. That way they don't have to be created from scratch for every test. Pending that, this looks good to merge to me!